app/views/layouts/application.html.erb

```diff
@@ -2,14 +2,15 @@
 <html>
   <head>
     <title><%= full_title(yield(:title)) %></title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render "layouts/rails_default" %>
+    <%= render "layouts/slim" %>
   </head>
   <body>
-    <%= link_to "English", locale:"en" %>
-    <%= link_to "Vietnamese", locale:"vi" %>
-    <%= yield %>
+    <%= render "layouts/header" %>
+    </header>
```

Comment:
sao lại có thẻ đóng header ở đây nhỉ?

Reply:
La the dong cua the phia tren a :  <%= render "layouts/header" %>

Reply:
sao e k cho thẻ đóng vào partial đó luôn?

Reply:
Em cho the dong vao roi ak

Reply:
Chi review cho em voi a